arch/mips/src/common/mips_initialize.c

```diff
@@ -67,10 +52,6 @@
 
 void up_initialize(void)
 {
-  /* Initialize global variables */
-
-  CURRENT_REGS = NULL;
```

Comment:
Just to make sure that this is intentional change

Reply:
Yes, the global variable is already zero.